xl-v1-metadata.go

```diff
@@ -52,6 +52,12 @@ type checkSumInfo struct {
 	Hash      string `json:"hash"`
 }
 
+// Default bit-rot checksum technique used in version '1'.
+const (
+	bitRotAlgoV1 = "blake2b"
```

Comment:
let's say we move to a different algo 'sha512', how will you make the changes in the names bitRotAlgoV1 and bitRotAlgo to accommodate the new algo?